print network name when rm successfully

allencloud opened the pull request:
Like `docker rm`, `docker volume rm`, when removing succefully, 
Docker CLI print the name.

So, this PR did:
1.print network name when rm successfully

allencloud committed:
print network name when rm successfully

Signed-off-by: allencloud <allen.sun@daocloud.io>

thaJeztah commented on api/client/network.go:
```diff
@@ -113,6 +113,7 @@ func (cli *DockerCli) CmdNetworkRm(args ...string) error {
 			status = 1
 			continue
 		}
+		fmt.Fprintf(cli.out, "%s\n", net)
```
Should this be `fmt.Fprintln(cli.out, net)`? (I see we use `Fprintf(` in other locations, just wonder why)

allencloud replied:
At this place, both are OK. Using a format in `fmt.Fprintln` will make the code more flexible.

thaJeztah replied:
I understand the differences, but [Fprintf](https://golang.org/src/fmt/print.go?h=doPrintf#L1101) does a lot more handling under the hood that's not needed here, so was wondering if we should just use println (given that we don't use any real formatting here). Sorry, just thinking out loud

allencloud replied:
Yes, that is true. While what I am thinking about is to keep consistency of all Docker CLI's code style. Almost all the CLI side code uses `Fprintf`.

thaJeztah replied:
Yeah, was thinking to change that for those places where it isn't needed. It's nit-picking probably, so don't let it block this PR 👍

thaJeztah commented:
LGTM

runcom commented:
LGTM

runcom commented:
Does `rmi` work the same by printing the removed image? Can't check right now

allencloud commented:
@runcom 
`docker rmi` will not output more details. It is more complicated than network, container and volume. First untag, then delete image if necesary.
```
root@ubuntu:~# docker rmi test:1
Untagged: test:1
Deleted: sha256:13872a98622642da207dff834675e6f031bd5ce74065a14dd7db45fdce5d002e
Deleted: sha256:4478325968ec99f91a834577aa30a76dd3da5d39f1c684d5df478844888c230a
Deleted: sha256:e8265bb1449667418f591bf4f6639c9ef3d1575e5a45376faeb8ff30957a3c57
Deleted: sha256:709e3d704ecd95a1e3db485ff714b9c529e7f49642360c983e719494654db9ec
Deleted: sha256:cc36a146915ea6669502825015d12d4bcb711397c5d8933ca7e7071193703e53
Deleted: sha256:3b709077ce051f4a44b6cef6c388abb18bcae71aae49d069cd8ee7638c824a18
Deleted: sha256:c298940f50bfcabad2db99ef745a7a57ecd0e8f6f27587e47c5a57008dc9d4c1
Deleted: sha256:224ac051c03239abbd6ffa533d36f673e3db80cd03fcbcdfe238a5ec77660357
Deleted: sha256:202858ed5145fb963779be903a2e67d9d2564793e9a9cbf507fd5503cf15cc1f
Deleted: sha256:cfe347175d9b6fa8cb412d9ad8e40ef577b27c5df0f4da5c0ca0a873fabdeafd
Deleted: sha256:44ab5a0119c3a3730a806590639d39ff364fad11842806aeb7c5e29491f765d2
Deleted: sha256:67a57e43789f767ad1b742b29ef90fd0d3989d5c2dfbd60bc123624db3529a47
Deleted: sha256:f3b401005eb17137c2db9c2e4bdb42636821894f9a12dbc6309eeaafd09e560d
Deleted: sha256:eb5e66221717ea49f301dcca915b617eccc6938e6b34474779bdf2f29002ed55
Deleted: sha256:f29cf682f60d803b0b462f4d2879a573ef31e3b8d28b73befa92d876ee19a575
Deleted: sha256:eaa96f844fd5618a40f7ffb0975b9f11483a1cbda8c329057418ba2741c7e842
Deleted: sha256:4439444b37ef3d5eb156ea7360b1f5a9edf0d9256b251451c46c3504acbd84f7
Deleted: sha256:682defc19e9327ecc080524f617bd05b24ab4388fbf35a187a5443f9b608b218
Deleted: sha256:51b7900587d2955da5690af1d3b8f85d691d2c3741a1ba1c534ea9639b56584f
Deleted: sha256:6acfa57070b1fe5ee6f07e95a646a0f7b85390b1c9e8afe8d1e7f9aeb1658aab
Deleted: sha256:6d57bc0a095c737680de9fd8b9273185b5bd60d6377670cdd888b6408f982604
Deleted: sha256:f66af1098d7188efa21a8b93946de7fa208fe98e1816c5079a8a3cb18e592411
Deleted: sha256:55c3fafc1852312193878460363ee585a3adf13e56808c95103111c406c04c54
```

runcom commented:
I see, it actually prints something so it makes sense to have this here, LGTM standing :)

vdemeester commented:
LGTM :angel:
